kudo-operator/params.yaml

```diff
@@ -92,6 +92,21 @@ ingressUrlFormat:
   default: ""
   displayName: "Ingress URL format"
 
+enableHistoryServer:
+  description: "Start Spark History Server"
+  default: true
+  displayName: "Enable Spark History Server"
+
+historyServerFsLogDirectory:
+  description: "Spark EventLog Directory from which to load events for prior Spark job runs (e.g., hdfs://hdfs/ or s3a://path/to/bucket). Note: You'd typically want to set this to point to distributed/HA storage"
+  default: "/opt/spark/work-dir"
+  displayName: "Spark History Server Event Log Directory"
+
+storageClassName:
+  description: "Storage class for provisioning a persistent volume used for Spark Event Logging."
+  default: "awsebscsiprovisioner"
```

Comment:
can we either rely on `DefaultStorageClass` admission controller or request this parameter from a user if `DefaultStorageClass` is not configured and `createPersistentVolumeClaim` is `true`? It would be really great to avoid binding to a single cloud provider so probably we can just make PVC creation external to KUDO Operator if there's no way to avoid explicit configuration?